hack/bootstrap-cluster.sh

```diff
@@ -2,7 +2,13 @@
 
 MODE=$1
 
-ROOT="$(realpath -mq ${BASH_SOURCE[0]}/../..)"
+
+if which realpath > /dev/null; then
+  ROOT="$(realpath -mq ${BASH_SOURCE[0]}/../..)"
+else
+  ROOT="$( cd "$(dirname "$0"/..)" >/dev/null 2>&1 ; pwd -P )"
```

Comment:
`ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/..` - does this work for you? If so, then please use it, it's used in other scripts. (No need to have that `if`)

Reply:
Yes, that works for me.  Updated

Reply:
Note however that the script never completes for me.  Have to ctrl-c.  

```
Installing the OpenShift GitOps operator subscription:
subscription.operators.coreos.com/openshift-gitops-operator created

Waiting for default project (and namespace) to exist: .....................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................^C
```